test/framework/service.go

```diff
@@ -18,7 +20,22 @@ const (
 
 func (f *Framework) doRequest(endpoint, method string, body []byte, query map[string]string) (respBody []byte, code int, err error) {
 	var u *url.URL
-	if f.UseKubeProxyForReportingAPI {
+
+	if f.UseRouteForReportingAPI {
+		const routeName = "metering"
```

Comment:
Can you put this at the top of the file under the imports like the other consts? And match the naming.